satpy/readers/nwcsaf_nc.py

```diff
@@ -75,21 +84,35 @@ def __init__(self, filename, filename_info, filetype_info):
 
         self.nc = self.nc.rename({'nx': 'x', 'ny': 'y'})
         self.sw_version = self.nc.attrs['source']
+
         self.pps = False
+        self.platform_name = None
+        self.sensor = None
 
         try:
-            # MSG:
-            sat_id = self.nc.attrs['satellite_identifier']
+            # NWCSAF/MSG:
```

Comment:
That should be NWCSAF/GEO, as it also covers ABI, AHI, and FCI.

Reply:
Right, fixed. Thx.